imageroot/actions/restore-module/71reload_services

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+
+#
+# Copyright (C) 2023 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+set -e
+
+sleep 5s
```

Comment:
why?

Reply:
There's no other way to check if freepbx container is healthy or ready at the moment.
The command following the sleep needs a running freepbx container, a podman call can be made (`podman wait --condition running freepbx`) to check if the container is running.

Multiple `fwconsole r` can interfere with one another?

Reply:
no should be ok having multiple reload. 
Could we create a file at the end of the entrypoint?